stl/inc/cmath

```diff
@@ -622,20 +681,20 @@ _GENERIC_MATH1(erf)
 _GENERIC_MATH1(erfc)
 _GENERIC_MATH1(lgamma)
 _GENERIC_MATH1(tgamma)
-_GENERIC_MATH1(ceil)
-_GENERIC_MATH1(floor)
+_GENERIC_MATH1I(ceil, __ceil)
+_GENERIC_MATH1I(floor, __floor)
 _GENERIC_MATH1(nearbyint)
 _GENERIC_MATH1(rint)
 _GENERIC_MATH1R(lrint, long)
 _GENERIC_MATH1R(llrint, long long)
-_GENERIC_MATH1(round)
+_GENERIC_MATH1I(round, __round)
 _GENERIC_MATH1R(lround, long)
 _GENERIC_MATH1R(llround, long long)
-_GENERIC_MATH1(trunc)
+_GENERIC_MATH1I(trunc, __trunc)
 _GENERIC_MATH2(fmod)
 _GENERIC_MATH2(remainder)
 // remquo() is hand-crafted
-_GENERIC_MATH2(copysign)
+_GENERIC_MATH2I(copysign, __copysign)

```

Comment:
I see there are no changes to test coverage, which should be fine because there are no changes to behavior here. Did you verify that we have at least one call to each of these functions somewhere in our test suites?

Reply:
Yes, it's covered by https://github.com/microsoft/STL/blob/a6c10429b6326aa92b3e410ed6a1f3fa13ffa571/tests/std/tests/VSO_0000000_instantiate_iterators_misc/test.cpp#L1907-L1991